contexts.go

```diff
@@ -0,0 +1,18 @@
+package process
+
+import "context"
+
+type healthKeyType struct{}
+
+var healthKey = healthKeyType{}
+
+func ContextWithHealth(ctx context.Context, health *Health) context.Context {
```

Comment:
odd duck out?

Reply:
Yeah... `WithHealth` wasn't an option so I had to come up with something! :)